moderators.md

```diff
@@ -0,0 +1,137 @@
+
+## Volunteering
+
+If you would like to volunteer as a [moderator](governance.md#moderator), please fill out [this web form](). The form asks you about yourself:
+
+1. What is your full name?
+1. What is your professional affiliation?
+1. What is your GitHub username?
+1. Please list links that point to your online presence.
+1. What is your level of experience in R package development?
+Which packages, if any, have you created or contributed to?
+1. What other ways, if any, are you involved in the R community?
+1. What are your areas of subject matter expertise?
+
+and the ways you would like to get involved:
+
+1. Have you read the [moderator documentation](moderators.md) and [all R-multiverse policies](policies.md)? (Required for moderators.)
+1. Are you prepared to enforce [all R-multiverse policies](policies.md)? (Required for moderators.)
+1. How many hours per month will you commit to reviewing [contributed packages](contributions.md)? (Any amount is okay.)
+1. How many hours per month will you commit to moderating [issues](https://github.com/r-multiverse/help/issues) and [discussions](https://github.com/r-multiverse/help/discussions)? (Any amount is okay.)
+
+## Documentation
+
+If you become a [moderator](governance.md#moderator), please submit a pull request the [website repository](https://github.com/r-multiverse/r-multiverse.github.io) to add yourself to the [moderator](governance.md#moderator) list on the [teams page](https://github.com/r-multiverse/r-multiverse.github.io/blob/main/team.md).
+Please include your full name, a description, your GitHub profile photo if available, and your professional affiliation if feasible.
+
+## Access
+
+If you become a [moderator](governance.md#moderator), an [administrator](governance.md#administrator) will add you to the [R-multiverse moderator team](https://github.com/orgs/r-multiverse/teams/moderators) on GitHub.
+this team grants you maintainer access to <https://github.com/r-multiverse/contributions> and triage access to <https://github.com/r-multiverse/help> and <https://github.com/r-multiverse/topics>.
+
+# Scope
+
+This section explains the privileges and responsibilities of [moderators](governance.md#moderator).
+All these privileges and responsibilities are also shared with the [administrators](governance.md#administrator).
+If you have a question, please post a discussion to <https://github.com/r-multiverse/help/discussions> or confidentially reach out to one of the current [administrators](team.md#administrators).
+
+## Contributions
+
+[Moderators](governance.md#moderator) review R package contributions to R-multiverse.
+The contribution process is described at [on the website](https://r-multiverse.org/contributors.html) and is governed by the official [R-multiverse review policy](review.md).
```

Comment:
```suggestion
[The contribution process is described at on the website](https://r-multiverse.org/contributors.html) and is governed by the official [R-multiverse review policy](review.md).
```

Reply:
Per accessibility

Reply:
Added in 5dc30e052e0d191ce6ea56f0f10ffbb85f32d7e7 (not sure what's happening with the GitHub web UI)